packages/mdl-auto-init/README.md

```diff
@@ -0,0 +1,130 @@
+# MDL Auto Init
```

Comment:
You can, you just have to access the component instance property on the element:

```js
document.querySelect('.mdl-checkbox[data-mdl-auto-init]').MDLCheckbox.checked = true;
```

I'll add make that clearer, and also outline how this module is provided purely for convenience for simple sites.